### DISCUSSION TITLE
Is there a "scan in progress" function in the API

### DISCUSSION THREAD

**Discussion by donotos** [gotson/komga#1789] [Q&A] [answered]
Hello.
I'm trying to find how to see if there is a library scan running in the API.
On the Webui we can see it with the animated yellow bar just bellow the komga logo
But i can't seem to find this particular parameter in the API.

**Comment by gotson** [comment, marked as answer]
Not in the REST API, but you can subscribe to server side events (SSE) to get updates on tasks and other events. There is no documentation, but you can check the [code of the webui](https://github.com/gotson/komga/blob/master/komga-webui/src/services/komga-sse.service.ts)

Resources:
- https://en.wikipedia.org/wiki/Server-sent_events
- https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events

**Reply by donotos** [reply]
Thanks, i'll check this out.
